contracts/optimistic-ethereum/libraries/wrappers/Lib_SafeExecutionManagerWrapper.sol

```diff
@@ -42,7 +42,7 @@ library Lib_SafeExecutionManagerWrapper {
     }
 
     /**
-     * Makes an ovmCALL and performs all the necessary safety checks.
+     * Makes an ovmDELEGATECALL and performs all the necessary safety checks.
```

Comment:
Seeing this makes me remember how I really don't like the phrase we use throughout this, "`performs all the necessary safety checks`".

I think this is a little confusing because the `OVM_SafetyChecker` is what really "checks safety".  What these functions are really doing is *implementing the equivalent logic to what the safety checker looks for*.

Maybe an alternative way to say it is something like "and takes the necessary precautions to guarantee the sandbox can't be escaped", with a little more detail explaining the above in the header.

Reply:
I addressed this by adding a descriptive header comment: 
```
/**
 * @title Lib_SafeExecutionManagerWrapper
 * @dev The Safe Execution Manager Wrapper provides functions which facilitate writing OVM safe 
 * code using the standard solidity compiler, by routing all its operations through the Execution 
 * Manager.
 * 
 * Compiler used: solc
 * Runtime target: OVM
 */
 ```
 
 Then just making the function comments verify brief, ie. `Performs a safe ovmCALL.`.